src/components/Hero/Default/index.module.scss

```diff
@@ -1,24 +1,34 @@
 @use '@scss/common' as *;
 
 .defaultHero {
-  margin-bottom: calc(var(--base) * -3);
+  position: relative;
+  padding-top: calc(var(--base) * 5.4);
+  padding-bottom: calc(var(--base) * 5.4);
 
-  @include large-break {
-    margin-bottom: calc(var(--base) * -2);
-  }
-
-  @include mid-break {
-    margin-bottom: calc(var(--base) * -1);
+  @include small-break {
+    padding-top: calc(var(--base) * 3);
+    padding-bottom: calc(var(--base) * 3);
   }
 }
 
-.pageLabel {
-  margin-bottom: calc(var(--base) * 1);
-  display: block;
+.label {
+  margin-bottom: calc(var(--base) * 2);
+
+  @include small-break {
+    margin-bottom: calc(var(--base) * 0.8);
+  }
 }
 
 .richText {
   @include small-break {
     margin-bottom: var(--base);
```

Comment:
@paulpopus The `row gap` coming from `.container` already gives enough spacing between the heading and the bottom paragraph in mobile. I would probably remove this `margin-bottom` here